src/main/resources/application.properties

```diff
@@ -1,8 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store_db?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=124533
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-
 spring.jpa.hibernate.ddl-auto=validate
```

Comment:
Why are you deleting this? You will be unable to connect db after this

spring.datasource.url=jdbc:mysql://localhost:3306/book_store_db?serverTimezone=UTC
spring.datasource.username=root
spring.datasource.password=124533